client/payment-methods/index.js

```diff
@@ -156,37 +156,41 @@ const PaymentMethods = () => {
 				} ) }
 			>
 				{ isUpeEnabled && (
-					<CardHeader
-						size={ null }
-						className="payment-methods-header"
-					>
+					<CardHeader className="payment-methods__header">
+						<h4 className="payment-methods__header__heading">
+							Payment methods
```

Comment:
I'll add the "early access" pill once https://github.com/Automattic/woocommerce-payments/pull/2420 is reviewed and merged